packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/ios/Flutter/development.xcconfig

```diff
@@ -1,7 +1,6 @@
 #include? "Pods/Target Support Files/Pods-Runner/Pods-Runner.debug-development.xcconfig"
 #include "Generated.xcconfig"
 
-FLUTTER_TARGET=lib/main_dev.dart
```

Comment:
Why the FLUTTER_TARGET is removed ?

Reply:
Patrol uses `--target` to provide an entry point for the integration test. Hardcoding the `FLUTTER_TARGET` here overwrites the one that comes from `Generated.xcconfig`, which in turn prevents the integration test from running, since the app starts with `lib/main_dev.dart` instead of `test/integration_test/tests/counter_test.dart`.

Removing the hardcoded value doesn't interfere with the run actions for the different flavors that are present in the rx_bloc_cli. But for command line builds it needs to be verified that a `--target` argument is provided with the command.

Another option I could think of is to create a separate flavor for the integration tests, where the `FLUTTER_TARGET` is not overwritten, so the tests can run.